modules/genomic_browser/jsx/tabs_content/files.js

```diff
@@ -151,12 +152,10 @@ class Files extends Component {
   formatColumn(column, cell, rowData, rowHeaders) {
     let reactElement;
     switch (column) {
-      case 'PSCID':
-        const url = `${this.props.baseURL}/${rowData.DCCID}/`;
-        reactElement = <td><a href={url}>{rowData.PSCID}</a></td>;
-        break;
-      case 'Subproject':
-        reactElement = <td>{this.state.data.subprojects[parseInt(cell)]}</td>;
+      case 'Name':
+        const url = `${this.props.baseURL}${rowData.Name}`;
```

Comment:
rowData.Name contains the path of the file. I have set my config for genomic file to be in `/data/genomics` and the request url is `https://localhost:4443/data/genomics/genomic_uploader/my_file.txt`. This does not work.

Reply:
Hi @xlecours, my latest commit modifies the code to remove the `gemonic_uploader` from the URL path that was being stored in the database. So your URL should be `https://localhost:4443/data/genomics/my_file.txt`.  Even with the correct path to the file I'm unable to download the file when testing on my development setup and I'm wondering if Loris has changed how files are downloaded. Would you happen to know? Otherwise I might not have my development setup correctly to download files where I'm setting the path to.